examples/peerconnection/serverless/logger.cc

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include "logger.h"
+
+void FileLogSink::OnLogMessage(const std::string& msg,
+                           rtc::LoggingSeverity severity,
+                           const char* tag) {
+  OnLogMessage(tag + (": " + msg), severity);
+}
+
+void FileLogSink::OnLogMessage(const std::string& msg,
+                           rtc::LoggingSeverity /* severity */) {
+  OnLogMessage(msg);
+}
+
+void FileLogSink::OnLogMessage(const std::string& message) {
+  FILE* log_file_ = fopen(LOG_FILEPATH.c_str(), "a");
```

Comment:
I suggest to open the file in constructor and close it in destructor that will be better for efficiency.